hal/armv7m/stm32/l4/timer.c

```diff
@@ -127,20 +136,25 @@ void timer_jiffiesAdd(time_t t)
 
 void timer_setAlarm(time_t us)
 {
-	time_t ticks = hal_timerUs2Cyc(us);
 	u32 setval;
 	spinlock_ctx_t sc;
+	time_t ticks = hal_timerUs2Cyc(us);
 
-	if (ticks > 0xffffu)
-		ticks = 0xffffu;
+	if (ticks > 0xffff0000U)
```

Comment:
braces should be used even on single line if

Reply:
Thx